sql/project_record_ids.sql

```diff
@@ -25,8 +25,31 @@ DROP TABLE IF EXISTS project_record_ids;
 SELECT * INTO project_record_ids 
 FROM _project_record_ids;
 
+
+
+/* Add stand-alone projects. This includes unmatched residential DOB projects, 
+as well as projects from sources that were excluded 
+from the non-DOB match process. */
+INSERT INTO project_record_ids
+SELECT array[]::text[]||record_id as project_record_ids
+FROM (
+	SELECT record_id::text from combined
+	WHERE no_classa = '0' OR no_classa IS NULL
```

Comment:
`no_classa` units is the number of Class A units?

Reply:
it's binary for '0' as having no class a units and '1' for having any units

Reply:
Got it, thanks so its actually "no class A units"